.redhat-ci.sh

```diff
@@ -4,14 +4,17 @@ set -xeuo pipefail
 # https://bugzilla.redhat.com/show_bug.cgi?id=1318547#c7
 mount --make-rshared /
 
-systemctl start docker
 
 if [ -f /run/ostree-booted ]; then
     if [ ! -e /var/tmp/ostree-unlock-ovl.* ]; then
         ostree admin unlock
     fi
+else
+    dnf install -y atomic python3-coverage
```

Comment:
It is simpler to debug on a cloud-image for certain test failure (least for me).  It really doesnt matter.  This is only for local testing for developers so we can close this if there is objection.